README.md

```diff
@@ -114,7 +115,8 @@ This project has adopted the [im-open's Code of Conduct](https://github.com/im-o
 Copyright &copy; 2021, Extend Health, LLC. Code released under the [MIT license](LICENSE).
 
 <!-- Links -->
+
 [git-version-lite]: https://github.com/im-open/git-version-lite
-[PowerShell Remoting over HTTPS with a self-signed SSL certificate]: https://4sysops.com/archives/powershell-remoting-over-https-with-a-self-signed-ssl-certificate
-[WSMan]: https://docs.microsoft.com/en-us/windows/win32/winrm/ws-management-protocol
-[WinRM]: https://docs.microsoft.com/en-us/windows/win32/winrm/about-windows-remote-management
\ No newline at end of file
+[powershell remoting over https with a self-signed ssl certificate]: https://4sysops.com/archives/powershell-remoting-over-https-with-a-self-signed-ssl-certificate
+[wsman]: https://docs.microsoft.com/en-us/windows/win32/winrm/ws-management-protocol
```

Comment:
The casing changed on the link text, but it seems work fine.

Reply:
Sorry about all those whitespace changes.  Aggressive extension formatting for me.